disaster5/src/Assets.cs

```diff
@@ -68,8 +68,13 @@ public static Texture Texture(string path)
             if (textures == null) textures = new Dictionary<string, Texture>();
             if (!textures.ContainsKey(path)) {
                 var texturePath = LoadPath(path);
+
+                var imgPtr = SDL2.SDL_image.IMG_Load(texturePath);
+                
+                if (imgPtr == IntPtr.Zero)
```

Comment:
this should return null, and the texture drawing code should have the error checking for null texture